src/Aspire.Hosting.Elasticsearch/ElasticsearchContainerImageTags.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Elasticsearch;
+
+internal static class ElasticsearchContainerImageTags
+{
+    public const string Registry = "docker.io";
+    public const string Image = "elasticsearch";
```

Comment:
```suggestion
    public const string Image = "library/elasticsearch";
```

We are explicit about image names.